src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -39,6 +41,19 @@ public class GitHubPackage : AsyncPackage
         {
             LogVersionInformation();
             await base.InitializeAsync(cancellationToken, progress);
+            
+            var packageSettings = await GetServiceAsync(typeof(IPackageSettings)) as IPackageSettings;
```

Comment:
Could we move this into an `InitializeLogging` method or something?

Reply:
👍